Bottom: bf38d805cecbb29ad4eeaf8fed937dbf9eeaa591
Top:    51cdc21140cba2d70c2ae10355c2ed5354963fe1
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 12:36:32 -0700

Refresh of dbg

---

diff --git a/src/preload.c b/src/preload.c
index aa1939d..e429ad6 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -890,7 +890,7 @@ pid_t fork(void)
 		goto lclose;
 
 	dfd = raccept(lfd, NULL, NULL);
-	printf("fork - raccept %d\n", dfd);
+	printf("fork - raccept %d %s\n", dfd, strerror(errno));
 	if (dfd < 0)
 		goto lclose;
